cmd/ciImageBuild.go

```diff
@@ -204,7 +204,7 @@ var ciImageBuildCmd = &cobra.Command{
 			extraImageTagString = fmt.Sprintf("--tag '%s:%s'", imageUrl, extraImageTag)
 		}
 		command := fmt.Sprintf("docker build --tag '%s:%s' %s -f '%s' %s", imageUrl, imageTag, extraImageTagString, dockerfile, buildPath)
-		pipedExec(command, debug)
+		pipedExec(command, "", "ERROR: ", debug)
```

Comment:
My test deployment still outputs the "ERROR: " prefix when buildling docker images: https://app.circleci.com/pipelines/github/wunderio/drupal-project-k8s/6126/workflows/405ae7ec-e6cf-49cb-8444-c8ad2dd32636/jobs/19621?invite=true#step-120-0_62

Probably because the prefix is defined in this pipedExec() call, should be removed right?

The test deployment does use the [silta cli build](https://app.circleci.com/pipelines/github/wunderio/drupal-project-k8s/6126/workflows/405ae7ec-e6cf-49cb-8444-c8ad2dd32636/jobs/19621?invite=true#step-103-0_7) from your branch.

Reply:
Thanks! I adjusted message, [works now](https://app.circleci.com/pipelines/github/wunderio/drupal-project-k8s/6129/workflows/06357b64-abee-427f-9c89-f1c2bb3bacb4/jobs/19634). 
Regarding testing - there's an issue with wgetting the "test" release - the most recent cli branch always overwrites test release and it was overwritten with "[optional repository for chart validation command](https://github.com/wunderio/silta-cli/actions/runs/8388834996)" already too.